File: SolicitudAyudaServer/SolicitudAyuda.Model/Entities/SolicitudAyuda.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace SolicitudAyuda.Model.Entities
{
    [Table("SolicitudesAyuda")]
    public class SolicitudAyuda
    {
        public int Id { get; set; }
        public int NumeroExpediente { get; set; }
        public string CedulaSolicitante { get; set; }
        
        public int MaestroId { get; set; }
        public Maestro Maestro { get; set; }

        public int SeccionalId { get; set; }
        public Seccional Seccional { get; set; }

        public string Celular { get; set; }
        public string TelefonoCasa { get; set; }
        public string TelefonoTrabajo { get; set; }
        public string Email { get; set; }

        public byte? QuienRecibeAyuda { get; set; }

        public DateTime FechaSolicitud { get; set; }
        public DateTime? FechaAprobacion { get; set; }
        public int TipoSolicitudId { get; set; }
        public TipoSolicitud TipoSolicitud { get; set; }

        public string OtroTipoSolicitud { get; set; }

        public List<RequisitoSolicitud> Requisitos { get; set; }
        public List<AdjuntosSolicitud> Adjuntos { get; set; }

        public List<AprobacionSolicitud> AprobacionesSolicitud { get; set; }
        public string Concepto { get; set; }
        public decimal MontoSolicitado { get; set; }
        public decimal? MontoAprobado { get; set; }
        public int UsuarioId { get; set; }
        public Usuario Usuario { get; set; }

        public int EstadoId { get; set; }
        public EstadoSolicitud Estado { get; set; }
        public string Direccion { get; set; }

        public int BancoId { get; set; }
        public Banco Banco { get; set; }
        public string NumeroCuentaBanco { get; set; }

        public bool? ActaNacimientoHijoHija { get; set; }
        public bool? CopiaCedulaPadreMadre { get; set; }
        public bool? ActaMatrimonioUnion { get; set; }
        public bool? EsJubiladoInabima { get; set; }
        public bool? EstadoCuenta { get; set; }
        public List<ComentarioSolicitud> Comentarios { get; set; }

        public SolicitudAyuda()
        {
            Requisitos = new List<RequisitoSolicitud>();
            Adjuntos = new List<AdjuntosSolicitud>();
            Comentarios = new List<ComentarioSolicitud>();
        }
    }
}

File: solicitudAyudasClient/src/app/model/Solicitud/solicitudAyuda.ts
export interface SolicitudAyuda {
     Id:number;
     NumeroExpediente: number;
     CedulaSolicitante:string;
     MaestroId: number;
     Maestro:string;
     Edad: number;
     Cargo:string;
     SexoMaestro:string;
     CategoriaId: number;
     SeccionalId:number;     
     Seccional:string;
     Celular:string;
     TelefonoCasa:string;
     TelefonoTrabajo:string;
     Email:string;
     QuienRecibeAyuda: number;
     FechaSolicitud:Date;
     FechaAprobacion:Date;
     FechaCreacion:Date;
     TipoSolicitudId:number;
     TipoSolicitud:string;
     OtroTipoSolicitud:string;
     concepto:string;
     MontoSolicitado:number;     
     MontoAprobado:number;
     UsuarioId:number;     
     EstadoId:number;
     Estado:string;     
     Direccion:string;
     BancoId:number;     
     Banco:string;
     NumeroCuentaBanco:string;
     ActaNacimientoHijoHija:boolean;
     CopiaCedulaPadreMadre:boolean;
     ActaMatrimonioUnion:boolean;
     EsJubiladoInabima:boolean;
     EstadoCuenta:boolean;     

     Requisitos: RequisitoSolicitud[];
     Adjuntos: AdjuntoSolicitud[];
     DatosAprobacion: DatoAprobacion[];
     ProcesadaPorUsuario: boolean;
     SubTipoSolicitud:string;

}

export interface RequisitoSolicitud {
     Id: number;
     FormName:string;
     PossibleValues:string;
     Descripcion:string;
     Value:string;
     Values: string[];
}

export interface AdjuntoSolicitud {
     Id: number;
     ContentType:string;
     SizeMB: number;   
     DisplayName:string;
}

export interface DatoAprobacion {
     EstadoId:number;
     Usuario:string;
     Fecha: Date;
     Estado:string;
     Comentario:string;
}